inquire/src/prompts/multicount/mod.rs

```diff
@@ -0,0 +1,427 @@
+mod action;
+mod config;
+mod prompt;
+#[cfg(test)]
+#[cfg(feature = "crossterm")]
+mod test;
+
+pub use action::*;
+
+use std::fmt::Display;
+
+use crate::{
+    config::get_configuration,
+    error::{InquireError, InquireResult},
+    formatter::MultiCountFormatter,
+    list_option::ListOption,
+    prompts::prompt::Prompt,
+    terminal::get_default_terminal,
+    type_aliases::Scorer,
+    ui::{Backend, MultiCountBackend, RenderConfig},
+    validator::MultiOptionValidator,
+};
+
+use self::prompt::MultiCountPrompt;
+
+#[cfg(feature = "fuzzy")]
+use fuzzy_matcher::{skim::SkimMatcherV2, FuzzyMatcher};
+#[cfg(feature = "fuzzy")]
+use once_cell::sync::Lazy;
+#[cfg(feature = "fuzzy")]
+static DEFAULT_MATCHER: Lazy<SkimMatcherV2> = Lazy::new(|| SkimMatcherV2::default().ignore_case());
+type CountedListOption<T> = (u32, ListOption<T>);
```

Comment:
What do you think about making this a struct? Thinking about the DX I'm not a fan of making developers use `item.0` and `item.1` for members, preferring `item.count` and `item.value` instead.

Reply:
No problem. I initially did it just to be as "close" as possible to what you already had. I'm happy to make it a struct and do it that way.